src/App.jsx

```diff
@@ -18,10 +19,12 @@ function App() {
       <header>
         <Header user={user} />
       </header>
-      <Routes>
-        <Route path="/" element={<Articles />}/>
-      </Routes>
-        
+      <div className="page">
```

Comment:
please aim to use semantic elements wherever possible. Could this `<div>` be a `<main>` instead?